core/internal/mocks/caller_subscriber.go

```diff
@@ -2,12 +2,9 @@
 
 package mocks
 
-import (
-	context "context"
-
-	eth "github.com/smartcontractkit/chainlink/core/eth"
-	mock "github.com/stretchr/testify/mock"
-)
+import context "context"
```

Comment:
Could you update your generator dependencies and redo these? We keep switching between import formats in mocks because everyone has different versions of those packages. (OR let me know if yours are the latest)

Reply:
Oops, thanks.

Reply:
I had `mockery` version `1.0.0` before and after `go get github.com/vektra/mockery/.../`. However, `go generate` still resulted in some differences specifically pertaining to the change to `models.Duration`, which are now the only contents of the git commit tagged as `go generate`.